src/components/interfaces/MOBILE_API.xml

```diff
@@ -2803,6 +2811,27 @@
             <description>True if the system can utilize the haptic spatial data from the source being streamed. If not included, it can be assumed the module doesn't support haptic spatial data'. </description>
         </param>
     </struct>
+
+    <struct name="RGBColor" since="5.0">
+        <param name="red" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+        <param name="green" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+        <param name="blue" type="Integer" minvalue="0" maxvalue="255" mandatory="true" />
+    </struct>
+
+    <struct name="TemplateColorScheme" since="5.0">
```

Comment:
No reason to move this struct.

Reply:
@jacobkeeler there is the reason to move `RGBColor`, and I just would like to keep it near the `TemplateColorScheme` because they are related to the same area. Should I move `TemplateColorScheme` back?

Reply:
Already taken care of